server/app/services/DateConverter.java

```diff
@@ -53,4 +68,8 @@ public String renderDate(Instant time) {
     ZonedDateTime dateTime = time.atZone(zoneId);
     return dateTime.format(DateTimeFormatter.ofPattern("yyyy/MM/dd"));
   }
+  /** Formats an {@link LocalDate} to a String. */
+  public String renderDate(LocalDate date) {
+    return date.format(DateTimeFormatter.ofPattern("yyyy/MM/dd"));
```

Comment:
suggestion (blocking): extract a constant for `DateTimeFormatter.ofPattern("yyyy/MM/dd")` and use it here and on L69